test/e2e/metrics_endpoint_with_path_test.go

```diff
@@ -0,0 +1,53 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
+	kitmetricpipeline "github.com/kyma-project/telemetry-manager/test/testkit/kyma/telemetry/metric"
+	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
+)
+
+var _ = Describe("Metrics Endpoint with Path", Label("metrics"), func() {
+	const (
+		path     = "/v1/mock"
+		endpoint = "metric-mock"
+	)
+
+	var endpointDataKey string
+
+	makeResources := func() []client.Object {
+		var objs []client.Object
+
+		metricPipeline := kitmetricpipeline.NewPipeline("mock-metric-endpoint-path").
+			WithProtocol("http").
+			WithOutputEndpoint(endpoint).WithEndpointPath(path)
+
+		pipelineName := metricPipeline.Name()
+		endpointDataKey = fmt.Sprintf("%s_%s", "OTLP_ENDPOINT", kitkyma.MakeEnvVarCompliant(pipelineName))
```

Comment:
Let's replace the pipeline name here manually with the env var variant instead of introducing a helper.